src/Compute/Compute/help/New-AzImageConfig.md

```diff
@@ -13,7 +13,7 @@ Creates a configurable image object.
 ## SYNTAX
 
 ```
-New-AzImageConfig [[-Location] <String>] [[-Tag] <Hashtable>] [[-SourceVirtualMachineId] <String>]
+New-AzImageConfig [[-Location] <String>] [[-EdgeZone] <String>] [[-Tag] <Hashtable>] [[-SourceVirtualMachineId] <String>]
```

Comment:
@asager Was this (and other) help document generated using PlatyPS module? 
This notation means its a mandatory parameter: `[[-EdgeZone] <String>]`
For optional parameters, it should be `[-EdgeZone <String>]`

Reply:
I had to manually edit this line because of some odd behavior I was getting with the autogeneration. 99% of the rest is autogenerated. I'll fix this